### PR TITLE
Remove linter-flow-plus occurrences

### PR DESCRIPTION
**Pull request by peduxe**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -62,13 +62,13 @@ module.exports =
           }
       }
   , activate(){
-      console.log('activating linter-flow-plus')
+      console.log('activating linter-flow')
```
Comment (steelbrain) [inline src/index.js]: Just drop this line

File: src/index.js [hunk 2]
```diff
@@ -62,13 +62,13 @@ module.exports =
...
 
       // getting custom value
-      cmdString = atom.config.get('linter-flow-plus.pathToFlowExecutable')
+      cmdString = atom.config.get('linter-flow.pathToFlowExecutable')
     }
   , deactivate(){
-      console.log('deactivating linter-flow-plus')
+      console.log('deactivating linter-flow')
```
Comment (steelbrain) [inline src/index.js]: and this one as well

Comment (peduxe) [inline src/index.js]: yup :D

**Comment by nmn** [comment]
Thanks

**Pull request merged**